config/foreman.migrations/20220406150414_foreman_google.rb

```diff
@@ -0,0 +1,2 @@
+answers['foreman::plugin::google'] = answers['foreman::compute::gce']
+answers.delete('foreman::compute::gce')
```

Comment:
`Hash#delete` returns the value so you can also write this as:
```suggestion
answers['foreman::plugin::google'] = answers.delete('foreman::compute::gce')
```